+++ evince-0.3.1/data/evince.desktop.in.in.orig	2005-05-21 20:18:15.000000000 +0100
--- evince-0.3.1/data/evince.desktop.in.in	2005-05-21 20:18:47.000000000 +0100
@@ -12,5 +12,5 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=evince
 X-GNOME-Bugzilla-Component=General
-Categories=GNOME;Application;Graphics;VectorGraphics;Viewer;
+Categories=GTK;GNOME;Graphics;Viewer;
 MimeType=@EVINCE_MIME_TYPES@
