diff -Nur evince-0.8.0.orig/data/evince.desktop.in.in evince-0.8.0/data/evince.desktop.in.in
--- evince-0.8.0.orig/data/evince.desktop.in.in	2007-03-12 22:03:24.000000000 +0000
+++ evince-0.8.0/data/evince.desktop.in.in	2007-03-14 10:08:48.000000000 +0000
@@ -9,11 +9,10 @@
 Terminal=false
 Type=Application
 Icon=evince
-NoDisplay=true
 X-GNOME-DocPath=
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=evince
 X-GNOME-Bugzilla-Component=general
 X-GNOME-Bugzilla-Version=@VERSION@
-Categories=GNOME;GTK;Graphics;VectorGraphics;Viewer;
+Categories=GTK;GNOME;Graphics;Viewer;
 MimeType=@EVINCE_MIME_TYPES@
