diff -aurN evince-0.2.1.orig/data/evince.desktop.in evince-0.2.1/data/evince.desktop.in
--- evince-0.2.1.orig/data/evince.desktop.in	2005-04-20 12:29:56.000000000 +0200
+++ evince-0.2.1/data/evince.desktop.in	2005-04-24 13:53:34.000000000 +0200
@@ -12,5 +12,5 @@
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=evince
 X-GNOME-Bugzilla-Component=General
-Categories=GNOME;Application;Graphics;VectorGraphics;Viewer;
+Categories=GTK;GNOME;Graphics;Viewer;
 MimeType=application/pdf;application/postscript;application/x-gzpostscript
