diff -urN evince-0.3.4.orig/data/evince.desktop.in.in evince-0.3.4/data/evince.desktop.in.in
--- evince-0.3.4.orig/data/evince.desktop.in.in	2005-08-18 09:43:18.463811504 +0200
+++ evince-0.3.4/data/evince.desktop.in.in	2005-08-18 09:44:30.697830264 +0200
@@ -9,10 +9,9 @@
 Terminal=false
 Type=Application
 Icon=postscript-viewer.png
-NoDisplay=true
 X-GNOME-DocPath=
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=evince
 X-GNOME-Bugzilla-Component=General
-Categories=GNOME;Application;Graphics;VectorGraphics;Viewer;
+Categories=GTK;GNOME;Graphics;Viewer;
 MimeType=@EVINCE_MIME_TYPES@
