diff -urN evince-0.3.4.orig/configure.ac evince-0.3.4/configure.ac
--- evince-0.3.4.orig/configure.ac	2005-08-17 19:24:23.525313000 +0200
+++ evince-0.3.4/configure.ac	2005-08-17 19:39:30.129488216 +0200
@@ -57,10 +57,12 @@
 GLIB_GENMARSHAL=`$PKG_CONFIG --variable=glib_genmarshal glib-2.0`  
 AC_SUBST(GLIB_GENMARSHAL)
 
-PKG_CHECK_MODULES([DBUS], [dbus-glib-1 >= $DBUS_GLIB_REQUIRED],
-	[enable_dbus=yes],[enable_dbus=no])
-AC_SUBST([DBUS_CFLAGS])
-AC_SUBST([DBUS_LIBS])
+dnl PKG_CHECK_MODULES([DBUS], [dbus-glib-1 >= $DBUS_GLIB_REQUIRED],
+dnl	[enable_dbus=yes],[enable_dbus=no])
+dnl AC_SUBST([DBUS_CFLAGS])
+dnl AC_SUBST([DBUS_LIBS])
+
+enable_dbus=no
 
 if test "x$enable_dbus" = "xno"; then
 	AC_MSG_WARN([DBUS support is disabled since dbus $DBUS_GLIB_REQUIRED or higher was not found])
