--- evince-0.1.6/configure.ac.wiget	2005-03-05 15:34:24.868628085 +0100
+++ evince-0.1.6/configure.ac	2005-03-05 15:46:37.530335700 +0100
@@ -90,7 +90,7 @@
 
 dnl check for GS version
 AC_MSG_CHECKING(for Ghostscript version...)
-GS_VERSION=`gs --version | head -n 1 | sed 's/\([[0-9]]*\)\.\([[0-9]]*\).*/\1/'`
+GS_VERSION=`gs --version 2>&1| head -n 1 | sed 's/\([[0-9]]*\)\.\([[0-9]]*\).*/\1/'`
 AC_MSG_RESULT(found $GS_VERSION)
 if test "$GS_VERSION" -lt "7"; then
 	AC_MSG_ERROR([You need Ghostscript version >= 7 in order to run evince])
