--- evince-0.7.2/data/evince.desktop.in.in~	2007-02-13 13:13:29.000000000 +0100
+++ evince-0.7.2/data/evince.desktop.in.in	2007-02-13 13:14:38.988426888 +0100
@@ -9,11 +9,10 @@
 Terminal=false
 Type=Application
 Icon=evince
-NoDisplay=true
 X-GNOME-DocPath=
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=evince
 X-GNOME-Bugzilla-Component=general
 X-GNOME-Bugzilla-Version=@VERSION@
-Categories=GNOME;GTK;Application;Graphics;VectorGraphics;Viewer;
+Categories=GTK;GNOME;Graphics;Viewer;
 MimeType=@EVINCE_MIME_TYPES@
