--- evince-3.14.0/configure.ac~	2014-10-11 19:48:53.000000000 +0200
+++ evince-3.14.0/configure.ac	2014-10-11 19:50:09.204494246 +0200
@@ -174,24 +174,6 @@
 fi
 
 SHELL_PLATFORM_PKGS=
-case "$with_platform" in
-  gnome)
-        # Evince has a rather soft run-time dependency on hicolor-icon-theme.
-        # If the hicolor theme is not available, Evince fails to display some
-        # icons. Because we cannot check for it at run-time, we instead
-        # would like to require the icon theme at compile-time. But, because
-        # the hicolor-icon-theme does not have a pkgconfig file, on gnome we
-        # require the gnome icon theme instead.
-        SHELL_PLATFORM_PKGS="adwaita-icon-theme >= $ADWAITA_ICON_THEME_REQUIRED"
-        ;;
-  *)
-        # On all other platforms we issue a warning about the runtime
-        # dependency.
-        AC_MSG_WARN([Evince has a soft run-time dependency on hicolor-icon-theme. You are advised to have this theme installed when running Evince.]);
-        SHELL_PLATFORM_PKGS=""
-        ;;
-esac
-
 PKG_CHECK_MODULES([SHELL_CORE],[libxml-2.0 >= $LIBXML_REQUIRED gtk+-3.0 >= $GTK_REQUIRED gio-2.0 >= $GLIB_REQUIRED gmodule-no-export-2.0 >= $GLIB_REQUIRED gthread-2.0 $SHELL_PLATFORM_PKGS])
 
 # ***************
