--- evince-3.38.0/meson.build.orig	2020-09-12 21:36:51.826925800 +0200
+++ evince-3.38.0/meson.build	2020-09-20 10:38:50.641170542 +0200
@@ -171,13 +171,6 @@
 
 ev_platform = get_option('platform')
 if ev_platform == 'gnome'
-  # Evince has a rather soft run-time dependency on hicolor-icon-theme.
-  # If the hicolor theme is not available, Evince fails to display some
-  # icons. Because we cannot check for it at run-time, we instead
-  # would like to require the icon theme at compile-time. But, because
-  # the hicolor-icon-theme does not have a pkgconfig file, on gnome we
-  # require the gnome icon theme instead.
-  adwaita_icon_theme_dep = dependency('adwaita-icon-theme', version: '>= 2.17.1')
 
   # *** Nautilus property page build ***
   enable_nautilus = get_option('nautilus')
--- evince-3.38.0/shell/meson.build.orig	2020-09-12 21:36:51.862926000 +0200
+++ evince-3.38.0/shell/meson.build	2020-09-20 10:41:51.863522110 +0200
@@ -92,8 +92,6 @@
     output: resource,
     command: [find_program('windres'), '@INPUT@', '-I', data_dir, '@OUTPUT@'],
   )
-else
-  deps += adwaita_icon_theme_dep
 endif
 
 if enable_dbus
